### PR TITLE
Add Anime365 [RU] (fix #2376)

### LINKED ISSUE #2376: [Website Request] smotret-anime
### Website name

smotret-anime

### Website link

https://smotret-anime.com/

### Website language(s)

ru-RU

### Other details

Mirror: https://anime365.ru/
Also has NSFW mirror with completely same UI and abilities, `hentai365.ru`
Not sure about how hard would it be to implement it's support, since it's paywalled..

### PR DESCRIPTION
**Pull request by NoPlagiarism**
Anime365 is Russian pay-walled anime website

- ~~**PR is draft, cuz favicon.ico is being protected by Cloudflare**~~
- Has 3 domains. `smotret-anime.com` is less restricted to unauthorized users
- Data being scraped from MAL / AniDB and etc. So in theory all of anime pages should have link to MAL

### CODE REVIEW

**Review by lolamtisch — CHANGES_REQUESTED** [review]

File: src/pages/Anime365/tests.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Anime365",
+  "url": "https://smotret-anime.com",
+  "testCases": [
+    {
+      "url": "https://smotret-anime.com/catalog/genres",
```
Comment (lolamtisch) [inline src/pages/Anime365/tests.json:6]: Use an anime overview page. sync false in this case mean that it is a overview page

**Review by lolamtisch — CHANGES_REQUESTED** [review]

File: src/pages/Anime365/main.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      if (MalHref) {
+        return MalHref;
+      }
+      return false;
+    },
+    uiSelector(selector) {
+      let resSelector = '<div class="card"><div class="card-content">';
+      resSelector += selector;
+      resSelector += '</div></div>';
+      j.$('.body-container > .section > .container > .row > .col:last-of-type > .card')
```
[the remark this answers is not in the document]
Comment (NoPlagiarism) [reply src/pages/Anime365/main.ts]: Shortened selector

File: src/pages/Anime365/main.ts [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  },
+  init(page) {
+    api.storage.addStyle(
+      require('!to-string-loader!css-loader!less-loader!./style.less').toString(),
+    );
+    utils.urlChangeDetect(() => {
+      utils.waitUntilTrue(
+        () => {
+          return (
+            j.$('.body-container > .section > .container > .row > .col:last-of-type > .card')
```
[the remark this answers is not in the document]
Comment (NoPlagiarism) [reply src/pages/Anime365/main.ts]: Ok, used another element as detector, looks legit

**Review by lolamtisch — CHANGES_REQUESTED** [review]
Looks good. Now only the test needs to be fixed

**Pull request merged**